src/tools/fuzzing.h

```diff
@@ -257,12 +257,32 @@ class TranslateToFuzzReader {
     wasm.memory.exists = true;
     // use one page
     wasm.memory.initial = wasm.memory.max = 1;
-    // init some data
-    wasm.memory.segments.emplace_back(builder.makeConst(Literal(int32_t(0))));
-    auto num = upTo(USABLE_MEMORY * 2);
-    for (size_t i = 0; i < num; i++) {
-      auto value = upTo(512);
-      wasm.memory.segments[0].data.push_back(value >= 256 ? 0 : (value & 0xff));
+    if (features.hasBulkMemory()) {
+      size_t memCovered = 0;
+      // need at least one segment for memory.inits
+      size_t numSegments = upTo(8) + 1;
+      for (size_t i = 0; i < numSegments; i++) {
+        Memory::Segment segment;
+        segment.flags = upTo(3);
```

Comment:
Maybe at least in a comment write the two names of the flags here, so it's less of an odd hardcoded constant.

Also, this should be `4` I think, since we want 0, 1, 2, 3?

Reply:
Nope, the only valid flags are 0, 1, and 2. It is not valid to be passive and have a memory index. I will change this to use bools and add an explanatory comment, though.